src/widgets/orientablepushbutton.cpp

```diff
@@ -0,0 +1,87 @@
+// Copyright(c) 2017-2019 Alejandro Sirgo Rica & Contributors
```

Comment:
Can you help me understand why this is an orientable push button instead of a normal push button? I don't see where the orientation is every changed so it might be better to make it a simple push button.

Reply:
https://github.com/flameshot-org/flameshot/pull/631/files#diff-735729613df2a40b1fc629636c6e7b6bR619